### PR TITLE
Add iteration to CSSTransformValue

### PR DESCRIPTION
**Pull request by wilddamon**
This PR is dependent on https://github.com/css-typed-om/typed-om/pull/119. I'm putting this up so I can keep track of it, but don't review it until that one is merged.

### CODE REVIEW

**Comment by wilddamon** [comment]
This is now ready for review.

**Comment by rjwright** [comment]
LGTM :)

File: test/js/css-transform-value.js [hunk 1]
```diff
@@ -120,4 +120,82 @@ suite('CSSTransformValue', function() {
...
+    var entries = [];
+    var iterator = transform.entries();
+    var entry = iterator.next();
+    while (!entry.done) {
+      entries.push(entry.value);
+      entry = iterator.next();
+    }
+    assert.deepEqual(entries, expectedEntries);
+    // for..of
+    var forOfEntries = [];
```
Comment (samthor) [inline test/js/css-transform-value.js]: Didn't you include a helper for this in `test-util.js`?

Comment (wilddamon) [reply test/js/css-transform-value.js]: Done.

**Pull request merged**